src/pages/ProjectPatches.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useQuery } from "@apollo/client";
+import { useLocation, useParams } from "react-router-dom";
+import {
+  PatchesPage,
+  getPatchesInputFromURLSearch,
+} from "components/PatchesPage";
+import { pollInterval } from "constants/index";
+import {
+  ProjectPatchesQuery,
+  ProjectPatchesQueryVariables,
+} from "gql/generated/types";
+import { GET_PROJECT_PATCHES } from "gql/queries";
+import { useNetworkStatus } from "hooks";
+
+export const ProjectPatches = () => {
+  const { id: projectId } = useParams<{ id: string }>();
+  const { search } = useLocation();
+  const patchesInput = getPatchesInputFromURLSearch(search);
+  const { data, startPolling, stopPolling, error } = useQuery<
+    ProjectPatchesQuery,
+    ProjectPatchesQueryVariables
+  >(GET_PROJECT_PATCHES, {
+    variables: { projectId, patchesInput },
+    pollInterval,
+    fetchPolicy: "cache-and-network",
+  });
+  useNetworkStatus(startPolling, stopPolling);
+
+  return (
+    <PatchesPage
+      pageTitle={`${data?.project.displayName ?? ""} Patches`}
```

Comment:
Since you are referring to the project value twice, and there are 3 layers of nesting. This might be cleaner destructured.

```
const {project} = data

project.displayName
project.patches
```